scripts/annotation.js

```diff
@@ -14,12 +14,12 @@ $.urlParam = function (a) {
 
 //Our stuff
 $(document).ready( ()=> {
-    const name = $.urlParam('name');
     const structure = $.urlParam('structure');
     let index = $.urlParam('index');
+    let name = $.urlParam('name');
     let image_div = $("#image");
     let choice;
-    $("#name").text(name);
+    $("#user").text(name);
```

Comment:
If we're going to change the div, we should change the variable name to `user` as well. But also later :) 
Why change name from `const` to `let`? do you expect it to change?